### ISSUE TITLE
OmniStudio Components support

### ISSUE THREAD

**Issue by gitusr246** [forcedotcom/devops-center-roadmap#93]
Would like to see OmniStudio Components deployment Support which will enable SF Industries customers to adopt DevOps Center easily.
More or less Vlocity Managed Package deployment support would be a great thing!!

**Comment by JoshClifton** [comment]
Support for industries is really needed as the native deployment tool is not the easiest to use. DevOps center allowing for these types of deployments would probably increase industries users adoption greatly

**Comment by axbloomf** [comment]
This definitely needed.  This is critical.

**Comment by sainath36999** [comment]
This definitely needed. This is critical.

**Comment by JasonL-HealthTrust** [comment]
Agreed, this would be extremely useful for us. We use Omnistudio components heavily and right now, deployment for them is difficult and messy.

**Comment by chrisfani** [comment]
Is there an update on this one?  As I understand it, the Vlocity / Omnistudio components have been merged into the Core platform which should open up the ability for DoC to support the Omni metadata
